### PR TITLE
Adds missing mutation toxins and recipes

### PR DESCRIPTION
**Pull request by USB3pt0**
## About The Pull Request

Adds Apid and Ethereal Mutation Toxin and their recipes to the game.

Apid Mutation Toxin requires 5 units of Unstable Mutation Toxin and 20 units of Honey.
Ethereal Mutation Toxin requires 5 units of Unstable Mutation Toxin and 20 units of Liquid Electricity.

## Why It's Good For The Game

Every other race has its mutation toxin, and now so do apids and ethereals.

## Changelog
:cl: novaremnant
add: Added Apid Mutation Toxin
add: Added recipe for Apid Mutation Toxin
add: Added Ethereal Mutation Toxin
add: Added recipe for Ethereal Mutation Toxin
/:cl:

### CODE REVIEW

**Comment by Penwin0** [comment]
Remember to add it to the wiki if/when it gets merged

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -741,14 +747,13 @@
...
 
-
-//////////////Mutatuion toxins made out of advanced toxin/////////////
-
 /datum/chemical_reaction/mutationtoxin/skeleton
 	name = /datum/reagent/mutationtoxin/skeleton
 	id = /datum/reagent/mutationtoxin/skeleton
 	results = list(/datum/reagent/mutationtoxin/skeleton = 5)
-	required_reagents  = list(/datum/reagent/aslimetoxin = 5, /datum/reagent/consumable/milk = 30, /datum/reagent/toxin/acid/fluacid = 30) //Because acid melts flesh off.
+	required_reagents  = list(/datum/reagent/mutationtoxin/unstable = 5, /datum/reagent/consumable/milk = 30, /datum/reagent/toxin/acid/fluacid = 30) //Because acid melts flesh off.
```
Comment (Penwin0) [inline code/modules/reagents/chemistry/recipes/others.dm]: Non documented change 😔

Comment (USB3pt0) [reply code/modules/reagents/chemistry/recipes/others.dm]: oh whoops.  That's from a different pull request, https://github.com/BeeStation/BeeStation-Hornet/pull/4177.

How would I go about splitting them?

Comment (Penwin0) [reply code/modules/reagents/chemistry/recipes/others.dm]: Yeah your other pull has the apid stuff as well, you should close one of the two and redo the changes in different branches

Comment (USB3pt0) [reply code/modules/reagents/chemistry/recipes/others.dm]: I'll just nix the skelly stuff for now and leave the Apid stuff in.

**Review by qwertyquerty — APPROVED** [review]

**Review by utilbird — APPROVED** [review]

**Pull request merged**